etcdmain/config.go

```diff
@@ -241,6 +241,9 @@ func newConfig() *config {
 	fs.StringVar(&cfg.ec.AuthToken, "auth-token", cfg.ec.AuthToken, "Specify auth token specific options.")
 	fs.UintVar(&cfg.ec.BcryptCost, "bcrypt-cost", cfg.ec.BcryptCost, "Specify bcrypt algorithm cost factor for auth password hashing.")
 
+	// gateway
+	fs.BoolVar(&cfg.ec.EnableGRPCGateway, "enable-grpc-gateway", true, "Enable GRPC gateway.")
```

Comment:
@mitake hi Hitoshi, this works fine with command line  but when specify`config-file` it is not working, unless, I think, user explicitly set it. Since we set it to true always for command line config, why not keep it true by default for both cmd and config file config? I will create a PR for discussion. Here is a related issue https://github.com/etcd-io/etcd/issues/12093 Thanks!

Reply:
@mitake I have created https://github.com/etcd-io/etcd/pull/12297 for discussion. Thanks!